client/my-sites/upgrades/checkout-thank-you/footer.jsx

```diff
@@ -6,44 +6,62 @@ import React from 'react';
 /**
  * Internal dependencies
  */
+import Button from 'components/button';
 import Card from 'components/card';
+import Gravatar from 'components/gravatar';
 import { isJetpackPlan } from 'lib/products-values';
+import sample from 'lodash/sample';
 
 const CheckoutThankYouFooter = React.createClass( {
 	propTypes: {
-		isDataLoaded: React.PropTypes.bool.isRequired,
-		receipt: React.PropTypes.object.isRequired
+		purchases: React.PropTypes.array
 	},
 
-	getText() {
-		if ( ! this.props.isDataLoaded ) {
-			return this.translate( 'Loading…' );
+	renderContactButton() {
+		let url = '/help/contact',
+			target = '';
+
+		if ( this.props.purchases && this.props.purchases.some( isJetpackPlan ) ) {
+			url = 'https://jetpack.me/contact-support/';
+			target = '_blank';
 		}
 
-		if ( this.props.receipt.data && this.props.receipt.data.purchases.some( isJetpackPlan ) ) {
-			return this.translate(
-				'Check out our {{supportDocsLink}}support docs{{/supportDocsLink}} ' +
-				'or {{contactLink}}contact us{{/contactLink}}.',
-				{
-					components: {
-						supportDocsLink: <a href={ 'http://jetpack.me/support/' } target="_blank" />,
-						contactLink: <a href={ 'http://jetpack.me/contact-support/' } target="_blank" />
-					}
-				}
-			);
+		return (
+			<a className="button" href={ url } target={ target }>
```

Comment:
Yes, I started with `<Button />` but switched to a simple anchor because I assumed that this component couldn't allow me to manage the `target` attribute. Looks like I was plain wrong.